packages/devextreme/js/__internal/scheduler/m_scheduler.ts

```diff
@@ -2438,7 +2438,7 @@ class Scheduler extends Widget<any> {
 
         return result;
       }),
-      targetElement: getPublicElement(target),
+      targetElement: target instanceof HTMLElement ? target : getPublicElement(target),
```

Comment:
minor:
```suggestion
      targetElement: getPublicElement($(target)),
```
it's common pattern in our code base, you can look up for `getPublicElement($(` in IDE

Also please consider moving this logic inside of `getPublicElement` to prevent similar mistakes in future